pyproject.toml

```
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit"]
-build-backend = "flit.buildapi"
+requires = ["flit_core"]
```

```suggestion
requires = ["flit_core  >=2,<4"]
```

I strongly encourage all packages to put version constraints ([see the docs](https://flit.readthedocs.io/en/latest/pyproject_toml.html#build-system-section)), so that it's practical to make breaking changes in `flit_core` from time to time. E.g. at some point the `[tool.flit.metadata]` table will go away in favour of the `[project]` table (PEP 621).